python/tk_multi_workfiles/browser_form.py

```diff
@@ -220,10 +245,50 @@ def set_models(self, my_tasks_model, entity_models, file_model):
             self._file_model.uses_user_sandboxes.connect(self._on_uses_user_sandboxes)
             self._file_model.set_users(self._file_filters.users)
 
-            # add an 'all files' tab:
-            self._add_file_list_form("All", "All Files", show_work_files=True, show_publishes=True )
-            self._add_file_list_form("Working", "Work Files", show_work_files=True, show_publishes=False)
-            self._add_file_list_form("Publishes", "Publishes", show_work_files=False, show_publishes=True)
+            # lowercase tabs to display as configured for the app
+            tabs_to_display = [
+                t.lower() for t in app.get_setting("file_browser_tabs")]
+
+            tab_count = 0
+
+            # iterate over each configured tab and see if it is valid (in the
+            # list of valid tabs)
+            for tab in tabs_to_display:
+                if tab in self.TAB_INFO.keys():
```

Comment:
I don't think you need the keys() call here. You can just do `tab in self.TAB_INFO`